sechub-pds-solutions/findsecuritybugs/docker-compose_pds_findsecuritybugs.yaml

```diff
@@ -5,7 +5,7 @@ services:
     pds-findsecuritybugs:
         build:
             args:
-                - BASE_IMAGE=debian:11-slim
+                - BASE_IMAGE=${BASE_IMAGE}
                 - PDS_VERSION=${PDS_VERSION}
```

Comment:
Please remove the `PDS_VERSION` variable: [#1862#issuecomment-1382224019](https://github.com/mercedes-benz/sechub/pull/1862#issuecomment-1382224019)